themes/default/content/case-studies/lykke.md

```diff
@@ -0,0 +1,67 @@
+
+## Challenges Faced
+
+Lykke originally built a monolithic stack with a wide variety of components. The engineering team wanted to adopt a serverless architecture and in April 2019, they decided to build a completely new infrastructure to support new applications - all without a dedicated ops team. To add to the complexity, the group needed to accurately model key parts of their infrastructure and create abstractions that would simplify scaling and make it accessible to both-back end and front-end engineers. This greenfield project was an exciting opportunity to select best-in-class solutions that worked well with their chosen cloud provider, AWS, and were compatible with their preferred engineering tools and frameworks -- React Native, .NET Core, and serverless architecture.
+
+## Solution: Pulumi Crosswalk and Infrastructure-as-Code
+
+Lykke engineering head Damian Hickey had been aware of Pulumi since the company launched in 2017 and decided to deploy the new infrastructure using [Pulumi Crosswalk for AWS]({{< relref "/crosswalk/aws" >}}). This allowed him to design and manage infrastructure within his development team, rather than building a dedicated ops team. Crosswalk also provided Lykke with built-in Well Architected patterns and best practices - helping the team move new applications into production faster.
+
+With Pulumi, Damian’s team quickly set up their infrastructure without having to learn a new, proprietary programming language. In addition to creating their own abstractions, models, and best-practice files, Lykke was also able to take advantage of pre-built templates and libraries available right out of the box.
+
+Lykke is now running multiple accounts with many core services each - all defined and managed with Pulumi. Pulumi Crosswalk allows the team to dynamically spin up instances through a single pull request within the stack and manage across multiple environments as needed.
+
+## Results
+
+One of the biggest considerations for the team was cost management. Pulumi helped Lykke adopt cloud-native architectures faster and migrate to managed services like Amazon Elastic Kubernetes Service (EKS) . This helped the Lykke team to run their services more efficiently, reducing costs by at least 20% and making their services more scalable.
+
+Pulumi Crosswalk also helped the team to configure services like Amazon VPC using only a dozen lines of code - something that required 10x more configuration with other IaC tools. The code review process also became faster, with infrastructure changes taking hours, rather than days to deliver.
```

Comment:
```suggestion
Pulumi also helped the team to configure services like Amazon VPC using only a dozen lines of code - something that required 10x more configuration with other IaC tools. The code review process also became faster, with infrastructure changes taking hours, rather than days to deliver.
```